etcdctl/ctlv2/ctl.go

```diff
@@ -54,6 +56,12 @@ func Start(apiv string) {
 		cli.StringFlag{Name: "cert-file", Value: "", Usage: "identify HTTPS client using this SSL certificate file"},
 		cli.StringFlag{Name: "key-file", Value: "", Usage: "identify HTTPS client using this SSL key file"},
 		cli.StringFlag{Name: "ca-file", Value: "", Usage: "verify certificates of HTTPS-enabled servers using this CA bundle"},
+		cli.BoolFlag{Name: "insecure-skip-tls-verify", Usage: "skip server certificate verification"},
+		cli.GenericFlag{Name: "cipher-suites",
+			Value: flags.NewStringSliceFlag(tlsutil.AvailableCipherSuites()...),
```

Comment:
does this mean AvailableCipherSuites() is the default value? I didn't think all available ciphers were included in the default set in go